src/App.tsx

```diff
@@ -9,7 +9,7 @@ import { theme } from '@pagopa/mui-italia';
 import UserRoute from 'routes/User';
 import { RouteHandleObject } from 'models/Breadcrumbs';
 import TransactionsList from 'routes/TransactionsList';
-import utils from 'utils';
+import utils from './utils';
```

Comment:
From 'utils' should be ok. This is becuase in tsconfig.json you cna find:

`"baseUrl": "src",`

Reply:
Ok, I've amended my last commit